src/logic/operate.js

```diff
@@ -0,0 +1,24 @@
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  let result = new Big(0);
+  const number1 = new Big(numberOne);
+  const number2 = new Big(numberTwo);
+  switch (operation) {
```

Comment:
Let's include support for modifier operator(%), which turns the number into a percentage based number, e.g. `95 % == .95`. [Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone4/performOperate.png).